src/fah/webclient/index.html

```diff
@@ -3,481 +3,465 @@
+          </div>
+          <div class="one-third column center">
+              <span class="share-text">Share:</span>
+              <span>
+                <a href="#" class="email"><img src="images/mail.png"/></a>
+              </span>
+              <span>
+                <a href="https://www.facebook.com/sharer.php?u=https%3A%2F%2Ffoldingathome.org&amp;t=Share+your+unused+computer+power+to+help+find+a+cure." class="facebook"><img src="images/facebook.png"/></a>
+              </span>
+              <span>
+                <a href="https://twitter.com/share?url=https%3A%2F%2Ffoldingathome.org&amp;text=Share+your+unused+computer+power+to+help+find+a+cure." class="twitter"><img src="images/twitter.png"/></a>
+              </span>
+          </div>
+          <div class="one-third column right">
+              <span><a href="#" class="btn share-join" id="btn-learn">Learn</a></span><span><a class="btn btn-news" href="https://foldingathome.org/news/">News</a></span><span><a class="btn btn-help" name="help" href="#">Help</a></span>
+          </div>
+      </div>
 
-              <div id="box-user">
-                <span>I'm folding as:<span>
-                <div class="hlarge" id="user-username"></div>
-                <div class="hlarge" id="user-team"></div>
-                <a href="#" id="btn-setupid">Change Identity</a>
+      <div class="row panel-top">
+        <div class="one-half column">
+          <div class="container">
+              <div class="row">
+                <div class="five columns">
+                  <div id="box-user">
+                    <div class="panel-top-heading">I'm folding as...</div>
+                    <div id="user-username" class="hlarge"></div>
+                    <div id="user-team" class="hlarge"></div>
+                    <a href="#" id="btn-setupid">Change Identity</a>
+                  </div>
+                </div>
+                <div class="seven columns">
+                  <div id="box-cause-selector">
+                    <div class="panel-top-heading">I support research fighting...</div>
+                    <select id="box-cause-id">
+                      <option value="ANY">Any disease</option>
+                      <option value="ALZHEIMERS">Alzheimer's</option>
+                      <option value="CANCER">Cancer</option>
+                      <option value="HUNTINGTONS">Huntington's</option>
+                      <option value="PARKINSONS">Parkinson's</option>
+                    </select>
+                  </div>
+                </div>
               </div>
-
+          </div>
+        </div>
+        <div class="one-half column">
+          <div id="box-points-loading">Loading...</div>
+          <div id="box-points">
+            <div class="panel-top-heading">Points earned</div>
+            <div>
+              <span id="box-points-counter">Unknown</span>
+              <a id="box-user-stats-link" href="#">(see stats)</a></span>
```

Comment:
I see no opening tag for the span attribute here
```suggestion
              <span><a id="box-user-stats-link" href="#">(see stats)</a></span>
```